fluent/fluent.go

```diff
@@ -49,6 +49,7 @@ type Config struct {
 	MaxRetryWait     int           `json:"max_retry_wait"`
 	TagPrefix        string        `json:"tag_prefix"`
 	Async            bool          `json:"async"`
+	AsyncStop        bool          `json:"async_stop"`
```

Comment:
`AsyncStop` seems something to specify to "stop eventually" to me, and misses to explain that events will be discarded.
How about using `ForceStopAsyncSend` or something else?
(I don't think that name is good enough. Different ideas are welcome :D)